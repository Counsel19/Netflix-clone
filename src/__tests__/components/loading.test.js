/* eslint-disable testing-library/no-node-access */
import React from "react";
import { render, screen } from "@testing-library/react";
import { Loading } from "../../components";

describe('<Loading />', () => { 
    it("renders the <loading /> components", () => {
        const { container } = render(
            <Loading src="/images/users/i.jpg" data-testid="loading" />
        );
        expect(screen.getByTestId('loading')).toBeTruthy();
        expect(screen.getByTestId('loading-picture')).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();
    });

    it("renders the <Loading.ReleaseBody />", () => {
        const { container } = render(
            <Loading.ReleaseBody />
        );
        expect(container.firstChild).toMatchSnapshot();
    });
 });