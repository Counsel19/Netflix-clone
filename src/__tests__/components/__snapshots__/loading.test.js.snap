// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders the <Loading.ReleaseBody /> 1`] = `null`;

exports[`<Loading /> renders the <loading /> components 1`] = `
<div
  class="loading__Spinner-sc-u0ddct-0 dVUlNc"
  data-testid="loading"
>
  <img
    class="loading__Picture-sc-u0ddct-1 jrXswB"
    data-testid="loading-picture"
    src="/images/users//images/users/i.jpg.png"
  />
</div>
`;
